internal-api/src/main/proto/systems/ajax/malov/internalapi/commonmodel/stock/short_stock.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+import "systems/ajax/malov/internalapi/commonmodel/stock/big_decimal.proto";
+
+package systems.ajax.malov.commonmodel.stock.short_stock.proto;
+
+option java_multiple_files = true;
+
+message ShortStockRecordResponseDto{
+  commonmodel.stock.big_decimal.proto.BDecimal openPrice = 1;
```

Comment:
do you actually need big decimals in your API? As I see, external service returns prices in Float :)
and I can't imagine situation when some stock will cost more than double type can hold and when we'll need more precision than cents

Reply:
We discussed to leave it as its